packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -179,32 +179,25 @@ class ChipThemeData with Diagnosticable {
   /// This will rarely be used directly. It is used by [lerp] to
   /// create intermediate themes based on two themes.
   const ChipThemeData({
-    required this.backgroundColor,
+    this.backgroundColor,
     this.deleteIconColor,
-    required this.disabledColor,
-    required this.selectedColor,
-    required this.secondarySelectedColor,
+    this.disabledColor,
+    this.selectedColor,
+    this.secondarySelectedColor,
     this.shadowColor,
     this.selectedShadowColor,
     this.showCheckmark,
     this.checkmarkColor,
     this.labelPadding,
-    required this.padding,
+    this.padding,
     this.side,
     this.shape,
-    required this.labelStyle,
-    required this.secondaryLabelStyle,
-    required this.brightness,
+    this.labelStyle,
+    this.secondaryLabelStyle,
+    this.brightness,
     this.elevation,
     this.pressElevation,
-  }) : assert(backgroundColor != null),
```

Comment:
I didn't think we were removing these yet: something about still asserting it in mixed-mode code.

Reply:
All of the theme properties are now null by default.

Reply:
Oh, I missed that bit.

Reply:
Duh.